keyboards/dotmod/dymium65/rules.mk

```diff
@@ -0,0 +1 @@
+ENCODER_MAP_ENABLE = yes 
```

Comment:
This is fixed on your "old" PR, why are you now introducing it back at the keyboard level?

Reply:
Thank you for reviewing this pr. And I am so sorry for the previous mistakes. I am new to this project and just noticed this option should be placed in the keymap level. Hope the new commit meets the requirements.

Reply:
> Overall this PR is objectively worse in many aspects than the other PR.
Could you please provide some details...? This will help me to understand and address any areas that need improvement.